rsv-metrics/libexec/probes/cacert-verify-probe

```diff
@@ -207,8 +207,8 @@ Additionally it makes sure that all IGTF CAs are installed.
     #working_dir = tempdir
 
     #Check if the file CA cers are installed from ITB or OSG, new or old format
-    local_urls = ["http://repo.grid.iu.edu/pacman/cadist/cacerts_sha256sum.txt",
-                  "http://repo.grid.iu.edu/pacman/cadist/cacerts_sha256sum-new.txt"]
+    local_urls = ["http://repo.opensciencegrid.org/pacman/cadist/cacerts_sha256sum.txt",
```

Comment:
This 404's (with both the grid.iu.edu and opensciencegrid.org addresses). I don't think we have to worry about the non-new hashes in EL6/EL7 so we should be able to drop it.